src/main/java/uk/gov/companieshouse/company/metrics/config/MongoTransactionConfig.java

```diff
@@ -18,10 +18,10 @@
 @EnableTransactionManagement
 public class MongoTransactionConfig extends AbstractMongoClientConfiguration {
 
-    @Value("${spring.data.mongodb.name}")
+    @Value("${spring.data.mongodb.metrics.name}")
```

Comment:
Shouldn't this be done for charges one as well? I know it will be read only but should test if this config working if you enable replica-set locally on docker